### PR TITLE
Draft: Update evolution to include delta & omicron

### PR DESCRIPTION
**Pull request by rando2**
### Checklist

- [ ] Text is formatted so that each sentence is on its own line.

- [ ] Pre-prints cited in this pull request have a GitHub issue opened so that they can be reviewed.

### CODE REVIEW

File: content/10.diagnostics.md [hunk 1]
```diff
@@ -98,15 +98,19 @@ Positive PCR results have also been reported for extended periods of time from s
...
+
 Additionally, the impact of viral evolution on RT-PCR sensitivity is a concern [@doi:10.1038/s41576-021-00360-w; @doi:10.2807/1560-7917.ES.2020.25.39.2001650].
 Using a panel that includes multiple targets can mitigate these effects [@doi:10.1016/j.ijid.2020.06.027].
+As an example, public health officials in the United States and the UK used PCR-based assays (ThermoFisher TaqPath COVID-19 assay) to identify sequences with a mutation (69-70del) associated with a variant of concern because the mutation interrupted binding of the qPCR probe [@doi:10.1101/2020.12.30.20249034].
+In the UK, one study found that this VOC (B.1.1.7) was present in more than 97% of diagnostic tests that returned negative for S-gene targets and positive for the other targets; thus, the frequency of S-gene target failure could be used as a proxy for the detection of B.1.1.7 [@url:https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/947048/Technical_Briefing_VOC_SH_NJL2_SH2.pdf; @url:https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/950823/Variant_of_Concern_VOC_202012_01_Technical_Briefing_3_-_England.pdf]._
```
Comment (agitter) [inline content/10.diagnostics.md:109]:
```suggestion
In the UK, one study found that this VOC (B.1.1.7) was present in more than 97% of diagnostic tests that returned negative for S-gene targets and positive for the other targets; thus, the frequency of S-gene target failure could be used as a proxy for the detection of B.1.1.7 [@url:https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/947048/Technical_Briefing_VOC_SH_NJL2_SH2.pdf; @url:https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/950823/Variant_of_Concern_VOC_202012_01_Technical_Briefing_3_-_England.pdf].
```

File: content/10.diagnostics.md [hunk 2]
```diff
@@ -98,15 +98,19 @@ Positive PCR results have also been reported for extended periods of time from s
...
+The FDA has highlighted that the performance of several diagnostic tests may be affected by the B.1.1.7 lineage because it could cause false negative tests [@url:https://www.fda.gov/medical-devices/letters-health-care-providers/genetic-variants-sars-cov-2-may-lead-false-negative-results-molecular-tests-detection-sars-cov-2; @individual-diagnostics].
```
Comment (agitter) [inline content/10.diagnostics.md:110]:
```suggestion
The FDA has highlighted that the performance of several diagnostic tests may be affected by the B.1.1.7 lineage because it could cause false negative tests [@url:https://www.fda.gov/medical-devices/letters-health-care-providers/genetic-variants-sars-cov-2-may-lead-false-negative-results-molecular-tests-detection-sars-cov-2].
```
We can probably remove the diagnostics reference now that the text is in that manuscript.

**Review by agitter — COMMENTED** [review]
A couple minor edits
